.travis.yml

```diff
@@ -32,19 +32,21 @@ install:
   - pipenv install --dev
 
 # perform testing:
-# --> run the test suite with pytest
+# --> run the test suite with pytest and codecov
+# --> Check for black format
 # --> lint the functions with flake8
 # --> lint the test suites with flake8
 # --> lint the functions with pylint
 # --> lint the test suites with pylint
 # --> lint the README documentation with mdl
+# --> Run django server
 script:
   - pipenv run pytest gatorgrouper/tests --cov-config pytest.cov --cov
-  - pipenv run black */**.py --check
-  - pipenv run flake8 tests*
-  - pipenv run flake8 */**.py
-  - pipenv run pylint tests
-  - pipenv run pylint */**.py
+  - pipenv run black */*/*.py --check
+  - pipenv run flake8 gatorgrouper/tests*
```

Comment:
These should also be changed, and the duplicates removed.

Reply:
I was wondering whether to keep them so that checks are split up to make it easier to find where a linting error lies.

Reply:
I think that regardless, the linting program will say which files are violating.